### ISSUE TITLE
Postfacto database

### ISSUE THREAD

**Issue by AX-DevOpsAdmin** [pivotal/postfacto#332]
Does Postfacto have a db folder to back up or not?

**Comment by textbook** [comment]
**TL;DR**: Postfacto **does** have a `db/` directory, but that **does not** require backing up (in most production uses).

- There's a `db/` directory for the Rails backend that contains seeds (`seeds.rb`), migrations (`migrate/`) and, in some cases, SQLite databases (`*.sqlite3`). These single-file DBs are for dev and test usage, they're not suitable for production use, and there aren't any in the `package.zip` of the app we distribute.
- The app itself is stateless, to allow it to be quickly restarted and scaled by platforms like [Tanzu Application Service](https://tanzu.vmware.com/application-service), so although you may want to retain a copy of the source code you deployed you won't need to back _any_ of those directories up on an ongoing basis.
- In production you should have a `DATABASE_URL` pointing Postfacto to the (MySQL or Postgres) database. This database _should_ be backed up, but the details of that will depend on where and how that database is hosted; it's outside of Postfacto's control.
